src/component/help/HelpButton.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './style.css';
+
+class HelpButton extends React.Component {
+    render(){
+        return(
+            <div class="ui button toolTip" data-tooltip="Click on a state to select it. Use the slider and radio buttons below to filter the data by date and type." data-position="right center"  data-variation="wide">
```

Comment:
Tentative description. Will be updated later. I'm not sure if we are going with hover or click.